docs/how-to-guides/use-mulitple-instance.md

```diff
@@ -0,0 +1,49 @@
+Using multiple instance can have some unintended behavior such as “duplicate events being fired”.
```

Comment:
@ashishmishra26 can we have a heading here? I'm not sure without a heading it'll look good.

Reply:
@gotoperson added